subsys/bluetooth/shell/bt.c

```diff
@@ -2141,6 +2141,13 @@ static int cmd_conn_phy_update(const struct shell *shell, size_t argc,
 	struct bt_conn_le_phy_param param;
 	int err;
 
+	if (default_conn == NULL) {
+		shell_error(shell,
+				"%s: at least, one connection is required\n",
```

Comment:
`\n` is redundant. `shell_error` adds in.

Reply:
Thanks @nordic-krch . 
@nordic-krch and @Thalley , after this PR, I will create another PR to add the null check for conn_update() and conn_data_length_update(), how about I fix "\n" at the next PR and we keep this approval process going?  Sound good to every approver?

Reply:
i think that you can remove `\n` in this PR, if there is no critical changes others reviewers don't need to re-review and approvals stays.

Reply:
I agree with @nordic-krch

Reply:
Roger, I've already done it.